src/app/core/common-components/entity-subrecord/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -94,7 +94,7 @@ export interface TableRow<T extends Entity> {
 })
 export class EntitySubrecordComponent<T extends Entity> implements OnChanges {
   @Input() isLoading: boolean;
-
+  @Output() filteredData: EventEmitter<any[]> = new EventEmitter<any[]>()
```

Comment:
what's the exact type of this? Is it `{ record: T}`? Or can we make it `T[]`?

Reply:
@sleidig, The Type is  `{ record: T}`